### PR TITLE
feat(mail): switched to smtp

### PR DESCRIPTION
**Pull request by LukasReindlGit**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -279,13 +279,12 @@ an oauth client will automatically be retrieved.
...
 
-Username / E-Mail for a G-Mail account that can be used to send out emails.
-
-    gmail_password:
-
-Password for the specified G-Mail account.
+    smtp_host:
+    smtp_port:
+    smtp_user:
+    smtp_password:
```
Comment (burnedikt) [inline README.md:287]: description per parameter missing.

Comment (LukasReindlGit) [reply README.md]: sry, didnt push after rebase

File: README.md [hunk 2]
```diff
@@ -279,13 +279,23 @@ an oauth client will automatically be retrieved.
...
-Username / E-Mail for a G-Mail account that can be used to send out emails.
+    smtp_host:
 
-    gmail_password:
+The host adress of the smtp server.
 
-Password for the specified G-Mail account.
+    smtp_port:
+
+The port used by the smtp server.
```
Comment (burnedikt) [inline README.md:290]:
```suggestion
The port used by the SMTP server.
```

File: README.md [hunk 3]
```diff
@@ -279,13 +279,23 @@ an oauth client will automatically be retrieved.
...
 ### Mail Setup
 
-    gmail_user:
+In order to send mails, smtp needs to be set up
 
-Username / E-Mail for a G-Mail account that can be used to send out emails.
+    smtp_host:
 
-    gmail_password:
+The host adress of the smtp server.
```
Comment (burnedikt) [inline README.md:286]:
```suggestion
The host adress of the SMTP server.
```

File: README.md [hunk 4]
```diff
@@ -279,13 +279,23 @@ an oauth client will automatically be retrieved.
 
 ### Mail Setup
 
-    gmail_user:
+In order to send mails, smtp needs to be set up
```
Comment (burnedikt) [inline README.md:282]:
```suggestion
In order to send mails, SMTP needs to be set up
```

File: README.md [hunk 5]
```diff
@@ -279,13 +279,23 @@ an oauth client will automatically be retrieved.
...
+The host adress of the smtp server.
 
-Password for the specified G-Mail account.
+    smtp_port:
+
+The port used by the smtp server.
+
+    smtp_user:
+
+The smtp user name.
```
Comment (burnedikt) [inline README.md:294]:
```suggestion
The SMTP user to authenticate with.
```

File: README.md [hunk 6]
```diff
@@ -279,13 +279,23 @@ an oauth client will automatically be retrieved.
...
+
+    smtp_password:
+
+The password corresponding to the given smtp user.
```
Comment (burnedikt) [inline README.md:298]:
```suggestion
The password corresponding to the given SMTP user.
```

**Pull request merged**